x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.tsx

```diff
@@ -460,8 +449,12 @@ export const TimelineDataTableComponent: React.FC<DataTableProps> = memo(
             showMultiFields={true}
             cellActionsMetadata={cellActionsMetadata}
             externalAdditionalControls={additionalControls}
-            renderCustomGridBody={renderCustomBodyCallback}
-            trailingControlColumns={trailingControlColumns}
+            renderCustomGridBody={
+              enabledRowRenderers.length > 0 ? renderCustomBodyCallback : undefined
```

Comment:
pull this check into a const?

Reply:
Did you mean `useMemo` because this cannot be moved out of the component as an independent const. But I have moved it to `useMemo` for consistency. Done here: 801c31448a2

Reply:
Yea...inside a memoized variable. Thanks!